lib/coverage-middleware.js

```diff
@@ -0,0 +1,45 @@
+var optionUtils = require('./blanket-options');
+var path = require('path');
+
+function loadBlanketOptions(options) {
+  var blanketOptions = null;
+  var optionsPath = path.join(options.root, 'tests', 'blanket-options');
+  try {
+    blanketOptions = require(optionsPath);
+  } catch (error) {
+    if (!optionUtils.isCurrentFormat(optionsPath + '.js')) {
+      throw new Error('blanket-options file is in old format - upgrade using `ember g ember-cli-blanket');
+    }
+  }
+  return blanketOptions;
+}
+
+function reporterForName(reporterName) {
+  return require(path.join(__dirname, 'reporters', reporterName + '-reporter'));
+}
+
+// TODO handle multiple reporters
+function loadReporter(blanketOptions) {
+  var reporterName = blanketOptions.cliOptions.reporters[0] || 'null';
```

Comment:
Use the first reporter listed or when none are present the `null` or `no-op` reporter